pkgs/development/web/publii/default.nix

```
@@ -0,0 +1,103 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+, makeShellWrapper
+, wrapGAppsHook
+, alsa-lib
+, at-spi2-atk
+, at-spi2-core
+, atk
+, cairo
+, cups
+, dbus
+, expat
+, glib
+, gtk3
+, libsecret
+, mesa
+, nss
+, pango
+, udev
+, xdg-utils
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "Publii";
```

i think we prefer lowercase package names?

```suggestion
  pname = "publii";
```